### PR TITLE
docs: added primitive

### LINKED ISSUE #349: Extended guide about primitives on docs
### Description

As a developer using TresJS I want to have better documentation about primitives so that I can understand it's functionality better

### Suggested solution

Add dedicated guide to primitives

### Alternative

_No response_

### Additional context

Requested by community

### Validations

- [X] I agree to follow this project's [Code of Conduct](https://github.com/Tresjs/tres/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/Tresjs/tres/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://tresjs.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/tresjs/tres/issues) that reports the same bug to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by alvarosabu**
Closes #349

### CODE REVIEW

File: docs/advanced/primitive.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Primitives
+
+The `<primitive />` component is a versatile  is a low-level component in TresJS that allows you to directly use any Three.js object within your Vue application without any abstraction. It acts as a bridge between Vue's reactivity system and Three.js's scene graph.
```
Comment (JaimeTorrealba) [inline docs/advanced/primitive.md:3]: is a versatile, low-level
an abstraction

File: docs/advanced/primitive.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+</template>
+```
+
+## Props
+
+`object`: This prop expects a Three.js Object3D or any of its derived classes. It is the primary object that the `<primitive />` component will render. In the updated example, a `Mesh` object with an associated `Material` is passed to this prop.
+
+## Usage with Models 
+
+The `<primitive />` component is speciallly usefull to render complex objects like models loaded from external sources. The following example shows how to load a model from a GLTF file and render it using the `<primitive />` component.
```
Comment (JaimeTorrealba) [inline docs/advanced/primitive.md:33]: useful
specially
for rendering

File: docs/examples/load-models.md [hunk 3]
```diff
@@ -96,7 +101,7 @@ import { OrbitControls, GLTFModel } from '@tresjs/cientos'
 
 This particular approach is more straightforward but gives you less control over the model.
 
-## useFBX
+## useFBX
 
 The `useFBX` composable is available from [@tresjs/cientos](https://github.com/Tresjs/tres/tree/main/packages/cientos) package.
```
Comment (JaimeTorrealba) [inline docs/examples/load-models.md:106]: from the ... package

**Review by JaimeTorrealba — CHANGES_REQUESTED** [review]

File: docs/advanced/primitive.md (hunk 1 above)
Comment (alvarosabu) [reply docs/advanced/primitive.md:3]: is not an abstraction

**Review by JaimeTorrealba — APPROVED** [review]

**Review by Tinoooo — APPROVED** [review]

**Pull request merged**